src/data-dictionary/events/AjaxRequest/timeToLastCallbackEnd.md

```diff
@@ -4,7 +4,8 @@ type: attribute
 units: seconds (s)
 events:
   - AjaxRequest
+  - BrowserInteraction
   - BrowserTiming
 ---
 
-The duration, in seconds, from the start of the request (timestamp) to the end of the last callback. This is not just an additive function; the callback time can overlap with the wait time.
\ No newline at end of file
+The duration, in seconds, from the start of the request (timestamp) to the end of the last callback. This is not just an additive function; the callback time can overlap with the wait time. This attribute is only seen in AjaxRequest Events that are tied to BrowserInteraction Events.
```

Comment:
The same applies to these attributes:
browserInteractionId, browserInteractionName, eventId, parentEventId, jsDuration, timeSinceBrowserInteractionStart, timeToSettle

I suppose the only ones that are not self-explanatory are jsDuration and timeToSettle. So, I would maybe add the same sentence to these two.